docs/_plugins/uid-permalinks.rb

```diff
@@ -0,0 +1,64 @@
+
+
+class PageUIDCollisionException < StandardError
+  def initialize(msg="Page UID hashes collided.")
```

Comment:
please add sufficient docstrings to describe the purpose of classes and methods! It should be easy to understand at a glance what this is file is doing, why it is needed and how it's doing it :)

Reply:
Moved the plugin to `tk-doc-generator` and added std header and docstrings.